src/demo/renderer.js

```diff
@@ -135,27 +135,39 @@ export const render = () => {
 
       if (state.appMode === AppMode.CreaturesVTrashDemo) {
         if (state.biasTextTime) {
+          if (
+            !state.canSkipPredict &&
+            $time() >= state.biasTextTime + timeBeforeCanSkipBiasText
+          ) {
+            setState({
+              canSkipPredict: true
+            });
+          }
+        }
+      } else if (state.isRunning) {
+        if (
+          !state.canSkipPredict &&
+          $time() >= state.runStartTime + timeBeforeCanSkipPredict
+        ) {
           setState({
-            canSkipPredict:
-              $time() >= state.biasTextTime + timeBeforeCanSkipBiasText
+            canSkipPredict: true
           });
         }
-      } else if (state.isRunning) {
-        setState({
-          canSkipPredict:
-            $time() >= state.runStartTime + timeBeforeCanSkipPredict
-        });
       }
 
       break;
     case Modes.Pond:
       drawPondFishImages();
 
-      setState({
-        canSkipPond: $time() >= currentModeStartTime + timeBeforeCanSkipPond,
-        canSeePondText:
-          $time() >= currentModeStartTime + timeBeforeCanSeePondText
-      });
+      if (
+        !state.canSkipPond &&
+        $time() >= currentModeStartTime + timeBeforeCanSkipPond
+      ) {
+        setState({canSkipPond: true});
+      }
+      if (!state.canSeePondText) {
```

Comment:
This shouldn't be just a simple flip should it? There's a timeout here.

Reply:
Oops, you're right!